setup.py

```diff
@@ -85,7 +85,6 @@
             },
         package_data=package_data,
         include_package_data = True,
-        test_suite='nose.collector',
```

Comment:
We could technically use pytest-runner here *but* setuptools upstream considers running `setup.py test` deprecated.

Reply:
Similar to the easy interface for testing after import, I don't think we need to provide ```setup.py test``` any longer.